tests/Unit/PointwiseFunctions/AnalyticSolutions/NewtonianEuler/Test_IsentropicVortex.cpp

```diff
@@ -115,10 +117,12 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.AnalyticSolutions.NewtEuler.Vortex",
 #endif
 }
 
-    // [[OutputRegex, The strength must be non-negative.]]
-    [[noreturn]] SPECTRE_TEST_CASE(
-        "Unit.PointwiseFunctions.AnalyticSolutions.NewtEuler.VortexStrength3d",
-        "[Unit][PointwiseFunctions]") {
+    // clang-format off
```

Comment:
Is clang-format indenting this line? It's fine if it is

Reply:
Yes, that's how clang-format indented this line.